src/com/sfuentes/DigitalClockTest.java

```diff
@@ -0,0 +1,181 @@
+package com.sfuentes;
+
+import java.util.ArrayList;
+
+public class DigitalClockTest {
+  public static void main(String[] args) {
+
+    generateTime_IncorrectHours();
+    generateTime_IncorrectMinutes();
+    generateTime_IncorrectSeconds();
+    generateTime_IncorrectTime();
+
+    generateTime_firstHour();
+    generateTime_CorrectMinutes();
+    generateTime_CorrectSeconds();
+    generateTime_lastHour();
+  }
+
+  //  Test Incorrect Hours
+  public static void generateTime_IncorrectHours() {
+    DigitalClock times = new DigitalClock();
```

Comment:
The class DigitalClock has only one method and it's static, so creating an object is not required.
As a suggestion please modify the DigitalClock class and remove static from generateHours method so it will make sense having to create an object of the class

Reply:
Resolved.